### DISCUSSION TITLE
Great Project! and i want to create similar project

### DISCUSSION THREAD

**Discussion by PARKER-X** [msiemens/tinydb#524] [Q&A]
I want to create similar project as a fun project for learning and I need some resources and advice how can i create similar. 
which module i can use other than dict like json,ujson ? Using another module makes it efficient? @msiemens

**Comment by msiemens** [comment]
Hi there! It's great to hear that you're interested in creating a project similar to TinyDB for learning purposes. With regards to data storage, there's a lot of options in Python, depending on the data format you want to store your data as. If you want to use JSON, you could use [`ojson`](https://github.com/ijl/orjson), which claims to be very fast. If you want other data formats, you can use text formats like TOML or YAML, but you could also create your own binary data format e.g. using Python's `cPickle` module.

Does this help you?
